store/models.py

```diff
@@ -30,3 +30,8 @@ def __str__(self):
         else:
             return f'{self.book.title} - Available'
 
+class Rating(models.Model):
+    book_id = models.IntegerField()
+    user_id = models.IntegerField()
+    rating = models.CharField(max_length=10,default='0')
```

Comment:
It is better to use `ForeignKey` relation, rather than directly storing the ids. Also, store the rating as an integer and add validators to that, so as to ensure that it lies between 0 and 10.